WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -22,14 +23,17 @@ class BookLibraryController: UIViewController {
         fatalError("init(nibName:bundle:) has not been implemented")
     }
     
-    var libraryViewModel: BookLibraryViewModel = {
```

Comment:
Why did you change the name? it's confusing because its a libraryViewModel, not a single bookDetailsViewModel